cpp/include/cudf/column/column_device_view.cuh

```diff
@@ -56,6 +56,10 @@ struct nullate {
   };
   struct NO : std::bool_constant<false> {
   };
+  /**
+   * @brief The`nullate::DYNAMIC` defers the assumption of nullability to runtime, rather at
+   * compiletime and the caller specifies if the column has nulls at runtime.
```

Comment:
```suggestion
   * @brief `nullate::DYNAMIC` defers the determination of nullability to run time rather than
   * compile time. The calling code is responsible for specifying whether or not nulls are
   * present using the constructor parameter at run time.
```